qualtran/bloqs/arithmetic/bitwise.py

```diff
@@ -50,33 +47,23 @@ def _cvs_converter(vv):
 
 
 @frozen
-class XorK(SpecializedSingleQubitControlledGate):
+class XorK(Bloq):
```

Comment:
Do we need the `_cvs_converter` defined above? It doesn't seem to be used in this file

Reply:
oops, removed